people-and-planet-ai/land-cover-classification/e2e_test.py

```diff
@@ -12,433 +12,211 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import csv
-import importlib
-import io
-import logging
-import os
-import platform
-import subprocess
-import sys
-from unittest.mock import Mock, patch
-import uuid
-
-import apache_beam as beam
-from apache_beam.io.filesystems import FileSystems
-from apache_beam.testing.test_pipeline import TestPipeline
-import ee
-import google.auth
-from google.cloud import aiplatform
-from google.cloud import storage
-import nbconvert
-import nbformat
-import numpy as np
+import tempfile
+import textwrap
+from typing import Callable
```

Comment:
For type hints (other than `Optional`), prefer `collections.abc`. See [PEP 585](https://peps.python.org/pep-0585/) for more info - tl;dr, it adds extra layers that are no longer necessary, especially py3.9 and onwards, other than for some uses like `Optional`.

```suggestion
from collections.abc import Callable
```